app/serializers/web_hook_topic_view_serializer.rb

```diff
@@ -0,0 +1,221 @@
+require_dependency 'pinned_check'
```

Comment:
This serializer is huge, how much of the payload does it actually reduce as compared to `TopicViewSerializer`?